molgenis-core-ui/src/main/resources/css/molgenis.css

```diff
@@ -39,13 +39,14 @@ html {
 
 body {
 	/* Margin bottom by footer height */
-	margin-bottom: 40px;
+    margin-bottom: 40px !important;
 }
 
 .footer {
 	position: absolute;
 	bottom: 0;
 	width: 100%;
+    font-size: 12px;
 	/* Set the fixed height of the footer here */
 	height: 40px;
```

Comment:
same as for body.margin footer , maybe add a comment if these values are linked.